cmd/agent/app/integrations.go

```
@@ -115,6 +121,59 @@ var showCmd = &cobra.Command{
 	RunE:  show,
 }
 
+func detectEnvironment() error {
+	rootDir, _ = executable.Folder()
+	for {
+		agentReleaseFile := filepath.Join(rootDir, reqAgentReleaseFile)
+		if _, err := os.Lstat(agentReleaseFile); err == nil {
+			reqAgentReleasePath = agentReleaseFile
+			break
+		}
+
+		parentDir := filepath.Dir(rootDir)
+		if parentDir == rootDir {
+			break
```

Looks to me like we should return an err here